.github/CODEOWNERS

```
@@ -77,6 +77,12 @@
 # NixOS integration test driver
 /nixos/lib/test-driver  @tfc
 
+# Systemd
+/nixos/modules/boot/systemd.nix      @NixOS/systemd @Kloenk
+/nixos/modules/boot/systemd-*.nix    @NixOS/systemd @Kloenk
+/nixos/lib/systemd-*.nix             @NixOS/systemd @Kloenk
+/pkgs/os-specific/linux/systemd      @NixOS/systemd @Kloenk
```

They don't have write access so there isn't any point in adding them here. I'd end up removing them as part of https://github.com/NixOS/nixpkgs/pull/162044 anyway.

```suggestion
/nixos/modules/boot/systemd.nix      @NixOS/systemd
/nixos/modules/boot/systemd-*.nix    @NixOS/systemd
/nixos/lib/systemd-*.nix             @NixOS/systemd
/pkgs/os-specific/linux/systemd      @NixOS/systemd
```

---

Done :+1: